proto/acl.proto

```diff
@@ -18,6 +18,6 @@ message ACL {
 
   message Permissions {
     bool read = 1;
-    bool write = 2;
+    bool      write = 2;
```

Comment:
**[staticcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>warning: code should be clang-formatted [-Wclang-format-violations]